purchase_stock_price_variance/models/stock_picking.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class Stockpick(models.Model):
+    _inherit = "stock.picking"
+
+    bypass_price_variance_check = fields.Boolean(
+        copy=False,
+        tracking=True,
+        groups="purchase_stock_price_variance.group_bypass_price_variance_check",
+        help="If enabled, no error is raised for price variance between "
+        "the product's standard price and purchase receipt unit price.",
+    )
+
+    def _action_done(self):
+        global_price_variance_threshold_percent = (
+            self.env.company.price_variance_threshold_percent
+        )
+        global_price_variance_threshold_amount = (
+            self.env.company.price_variance_threshold_amount
+        )
+        for pick in self:
+            if pick.picking_type_id.code != "incoming":
+                continue
```

Comment:
Can you please check if adding conditions using `_is_in()` and `_is_dropshipped()` for each stock move record is more appropriate instead of this?

Reply:
When I checked the odoo source code, checking `picking_type_id.code` is used in `stock`.`_is_in()` and `_is_dropshipped()` is introduced in `stock_account` and used them. In our module, stock_account is part of dependency because we use purchase_stock as a dependency. So, I think using `_is_in()` and `_is_dropshipped()` is more appropriate.